### PR TITLE
Change HTTP Status Codes to match documentation

### LINKED ISSUE #55: Create DELETE person endpoint
## Description
 
Create an endpoint for deleting a specific person from the database

## Acceptance Criteria

- [x] create a function called `deletePerson(personId)` in `person.controller.ts`
- [x] attach the `deletePerson(personId)` function to to `/person/<personID>` in `person.route.ts `

## Implementation details

Ensure any references to the person with the given <personID> are removed from any encounters or user tables.

## Testing notes

TBD

### PR DESCRIPTION
**Pull request by butterplup**
Fixes #55 and #54

Just some minor linting fixes as well as altering HTTP status codes to be in line with documentation and proper specifications.

### CODE REVIEW

File: backend/src/controllers/person.controller.ts [hunk 1]
```diff
@@ -122,57 +122,57 @@ export const deletePersons = async (
...
+      const emptyEncounters = deleteEncountersResult['array'];
+      const encountersBool = deleteEncountersResult['bool'];
+
+      // Delete person from current User document
+      const deleteUserResult = await userService.deleteUserPerson(id);
+
+      // Check all service function calls were valid
+      if (encountersBool && deletePersonsResult && deleteUserResult) {
+        // Make sure that empty encounters are also deleted from User
+        for (let i = 0; i < emptyEncounters.length; i + 1) {
```
Comment (sher812) [inline backend/src/controllers/person.controller.ts:155]: Not sure why but changing the i++ to i+1 seems to be causing some issues when attempting to delete multiple people from an encounter.

**Review by sher812 — CHANGES_REQUESTED** [review]

File: backend/src/controllers/person.controller.ts (hunk 1 above)
Comment (butterplup) [reply backend/src/controllers/person.controller.ts:155]: okay, will change back

**Review by sher812 — APPROVED** [review]
LGTM

**Review by sdiv877 — APPROVED** [review]
LGTM!

**Pull request merged**